tests/test.utils.js

```diff
@@ -37,6 +37,42 @@ describe('utils.singleLineString()', function() {
 });
 
 
+describe('utils.getVariable()', function() {
+  // This is the expected schema from eslint
+  var context = {
+    getScope: function() {
+      return {
+        variables: [{
+          name: 'foo',
+          defs: [{
+            type: 'Variable',
+            name: {
+              parent: {
+                init: {
+                  type: 'Literal',
+                  value: 'bar',
+                },
+              },
+            },
+          }],
+        }],
+      };
+    },
+  };
+
+  it('should return the correct variable in the given context.', () => {
+    var foo = utils.getVariable(context, 'foo');
+    assert.equal(foo.type, 'Literal');
+    assert.equal(foo.value, 'bar');
+  });
+
+  it('should return undefined if the variable doesn\'t exist.', () => {
```

Comment:
Use double quotes or a template string here to avoid  the need to escape the `'` in doesn't